src/datagrid/DataGrid.tsx

```diff
@@ -288,6 +291,7 @@ type DataGridState = {
     itemText: string;
     pagination?: DataGridPaginationState;
     isLoading: boolean;
+    showSelectAll: boolean;
```

Comment:
No need to keep `showSelectAll` in state.

Reply:
Yes